internal/services/containers/container_group_resource.go

```diff
@@ -553,6 +553,12 @@ func resourceContainerGroup() *pluginsdk.Resource {
 				ForceNew:     true,
 				ValidateFunc: keyVaultValidate.NestedItemId,
 			},
+
+			"key_vault_user_identity_id": {
```

Comment:
Currently the identity properties for CMK are inconsistent across the provider, some are called `identity_id`, `primary_user_assigned_identity_id`  or `user_assigned_identity_id`. We might want to consider standardising these for the next major release and having a `commonschema` for them like we do for identities.

All this to say, I think for the time being this might benefit from being called `key_vault_user_assigned_identity_id` since it's clear at first glance what type of ID it is and is also closer to what we currently have in the provider.

Reply:
Renamed.